windows/src/engine/keyman64/version.rc

```diff
@@ -16,10 +16,10 @@
         VALUE "CompanyName", KV_COMPANY_NAME
         VALUE "FileDescription", "Keyman Engine Processor x64\0"
         VALUE "FileVersion", KV_VERSION_STRING
-        VALUE "InternalName", "KEYMAN64\0"
+        VALUE "InternalName", "KEYMAN64-VER\0"
         VALUE "LegalCopyright", KV_LEGAL_COPYRIGHT
         VALUE "LegalTrademarks", KV_LEGAL_TRADEMARKS
-        VALUE "OriginalFilename", "KEYMAN64.DLL\0"
+        VALUE "OriginalFilename", "KEYMAN64-VER.DLL\0"
```

Comment:
Are the `-VER` intended here?

Reply:
Yes these where incorrect. However I can't see these as part of the project. Even for the current master setup.